qualang_tools/addons/variables.py

```diff
@@ -0,0 +1,12 @@
+from qm.qua import *
+
+
+def assign_variables_to_element(element, *variables):
```

Comment:
If I understand correctly, the usage would be:
`a = declare(fixed)`
`b = declare(int)`
 `assign_variables_to_element("qubit_0", a, b)`
`...`
`Real time computation with a and b`
And this will just add a 16ns latency at the beginning of the program to force a given element to be assigned to the variable. Is it correct?

Reply:
yes, exactly.